src/features/unified-share-modal/flowTypes.js

```diff
@@ -171,3 +172,123 @@ export type tooltipComponentIdentifierType =
     | 'shared-link-copy-button'
     | 'shared-link-settings'
     | 'shared-link-toggle';
+
+type BaseUSMProps = {
```

Comment:
This may be an opportunity to add additional sanity here. We could specify base types based on the USM feature they are for? some are for the invite collab section, while others are for shared link section, etc.

- functionality types would aid in future enhancements and refactoring/componentization 

Mostly an idea to help organize future use cases (assuming we need to introduce a way to disable some functionality in certain UIs)